chunkflow/flow/flow.py

```diff
@@ -1238,23 +1238,23 @@ def inference(tasks, name, convnet_model, convnet_weight_path, input_patch_size,
               default=False,
               help='default is doing maskout. ' +
               'check all zero will return boolean result.')
-@click.option('--affinity/--no-affinity',
+@click.option('--mask-affinity/--mask-image',
```

Comment:
From here the changes are just renaming affinity/no-affinity to mask-affinity/mask-image, they belong to the preview commit, aren't they?

Reply:
That's right, will work on it.

Reply:
Done.